go.mod

```diff
@@ -149,4 +149,4 @@ replace (
 	github.com/gogo/protobuf v1.3.1 => github.com/gogo/protobuf v1.3.2
 )
 
-replace github.com/cilium/ebpf => github.com/inspektor-gadget/ebpf v0.0.0-20230306203928-c17e33ba8bac
+replace github.com/cilium/ebpf => github.com/cilium/ebpf v0.0.0-20230320181651-5ea5368f9823
```

Comment:
We can completely remove the `replace` line and specify the commit in the `require` declaration above.